src/android/TP6mAggregate.jsx

```diff
@@ -112,58 +111,57 @@ async function pullAggregate({
           .select('value')
           .average();
       },
-    }
+    },
   );
   const result = window(
     { daily, g5Reference },
     {
       edges: ['test', 'platform', 'pushDate'],
-      value: row => {
+      value: (row) => {
         const { daily, g5Reference } = row;
 
         return round(
           selectFrom(daily, g5Reference)
             // IF NO REFERENCE VALUE FOR SITE, DO NOT INCLUDE IN AGGREGATE
             .map((d, r) => (missing(r) ? null : d))
             .geomean(),
-          { places: 4 }
+          { places: 4 },
         );
       },
-    }
+    },
   );
   const mask = window(
     { daily },
     {
       edges: ['test', 'platform', 'site'],
-      value: ({ daily }) =>
-        // if anything in the past week, then we will use the reference
-        !selectFrom(daily)
-          .reverse()
-          .limit(8) // 7+1 for the null entry
-          .exists()
-          .isEmpty(),
-    }
+      value: ({ daily }) => !selectFrom(daily)
+        .reverse()
+        .limit(8) // 7+1 for the null entry
+        .exists()
+        .isEmpty(),
+    },
```

Comment:
Does the new linting rules come with automatic removal of comments?

Reply:
I don't know.

Reply:
I see that the "conversation" view does not show the comment removed:

![image](https://user-images.githubusercontent.com/2334429/61900913-9497c500-aeec-11e9-9551-237f27897fa9.png)

did you remove it, or did the linter?

Reply:
The linter.